temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
```

Comment:
Refactor the function to use a dictionary mapping file types to their respective reset values for scalability and cleaner code. [medium]